store/organizations.js

```diff
@@ -109,15 +156,55 @@ const $actions = {
 
     if (organizations) commit(UPDATE_CACHE, organizations.records);
 
-    return organizations;
+    return _.cloneDeep(organizations);
+  },
+
+  async getOrganizationById({ commit, getters }, id) {
+    let organization = {};
+
+    if (getters.isOrganizationCached(id)) {
+      organization = getters.getOrganizationById(id);
+    } else {
+      organization = await this.$kronosApi.getOrganization(id);
+      if (organization) commit(UPDATE_CACHE, [ organization ]);
+    }
+
+    return _.cloneDeep(organization);
```

Comment:
missing allowReadFromCache=false
```suggestion
  async getOrganizationById({ commit, getters }, id, allowReadFromCache=false) {
    let organization = null

    if (allowReadFromCache && getters.isOrganizationCached(id)) {
      organization = getters.getOrganizationById(id);
    } else {
      organization = await this.$kronosApi.getOrganization(id);
      if (organization) commit(UPDATE_CACHE, [ organization ]);
    }

    return _.cloneDeep(organization);
```

Reply:
DONE! changes done as suggested.